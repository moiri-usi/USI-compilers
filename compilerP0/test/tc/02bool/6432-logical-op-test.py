print 2 and 3
print 2 or 3
print 2 & 3
print 2 | 3
print not 1
print not 0
