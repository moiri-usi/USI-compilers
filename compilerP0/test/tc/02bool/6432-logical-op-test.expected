3
2
2
3
False
True
cde
abc
